src/include/network/lan/address.rb

```diff
@@ -475,25 +475,11 @@ def InitBridge(key)
     # @param [String] key	id of the widget
     # @param [String] key id of the widget
     def StoreBridge(key, _event)
-      Ops.set(
-        @settings,
-        "BRIDGE_PORTS",
-        String.CutBlanks(
-          Builtins.mergestring(
-            Convert.convert(
-              UI.QueryWidget(Id("BRIDGE_PORTS"), :SelectedItems),
-              from: "any",
-              to:   "list <string>"
-            ),
-            " "
-          )
-        )
-      )
-      Builtins.y2milestone(
-        "store bridge %1 : %2",
-        key,
-        Ops.get_string(@settings, "BRIDGE_PORTS", "")
-      )
+      @settings["BRIDGE_PORTS"] = selected_bridge_ports.join(" ")
+
+      LanItems.bridge_ports = @settings["BRIDGE_PORTS"]
```

Comment:
Sure, and is how differs also in configuration per interface. As bonding master is configured as:
```
BONDING_SLAVE0='eth0'
BONDING_SLAVE1='eth1'
```
while bridge_ports are configured as 
```
BRIDGE_PORTS='eth0 eth1'
```